composer.json

```diff
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "shopware/core": "~6.5.0 || ~6.6.0"
+        "shopware/core": "^6.5.0"
```

Comment:
This should be either `~6.7.0` or if it still works as is with both versions: `~6.6.0 || ~6.7.0`

Reply:
this plugin is so simple, I would keep it compatible between 6.5-6.7 😅 , as the pipeline went green. seems to work :)

Reply:
But isn't 6.5 EOL anyway with the release of 6.7? And why should 6.5 receive further updates of SwagPlatformDemoData if not even 6.5 gets updates 🤔 

But yes theoretically we could keep the support for 6.5 for future updates until we depend on some newer shopware / PHP feature that isn't supported in that version 🤷‍♂️